windows-iotcore/TOC.md

```diff
@@ -1,6 +1,6 @@
-# [Windows 10 IoT documentation](index.md)
```

Comment:
so no link to index.md?

Reply:
Added it in. I wasn't sure if adding a link would suppress the supplementary navigation, but it's worth a shot.